images/router/haproxy/conf/haproxy-config.template

```diff
@@ -85,6 +85,10 @@ listen stats :1936
     mode http
     # Health check monitoring uri.
     monitor-uri /healthz
+{{ if (and .State .ServiceUnits) }}
+    # This uri being configured indicates that routing configuration has been synchronized.
+    monitor-uri /readyz
```

Comment:
What's with the z's?  Should we really use that?  I was being somewhat facetious on the call when I said it.